README.md

```diff
@@ -41,29 +41,7 @@ legally purchased devices and games and information made public on the internet
 
 ## Quickstart
 
-With Windows 8+, Python 3.4+, and [Visual Studio 2017 or 2019](https://www.visualstudio.com/downloads/) and the Windows SDKs installed:
-
-    > git clone https://github.com/xenia-project/xenia.git
-    > cd xenia
-    > xb setup
-
-    # Pull latest changes, rebase, and update submodules and premake:
-    > xb pull
-
-    # Build on command line:
-    > xb build
-
-    # Run premake and open Visual Studio (run the 'xenia-app' project):
-    > xb devenv
-
-    # Run premake to update the sln/vcproj's:
-    > xb premake
-
-    # Format code to the style guide:
-    > xb format
-
-When fetching updates use `xb pull` to automatically fetch everything and
-run premake for project files/etc.
+See the [Quickstart](https://github.com/xenia-project/xenia/wiki/Quickstart) page.
```

Comment:
I see what you're doing with the Quickstart, but this Quickstart for building instructions is still relevant. Please relocate it to the building doc.

Reply:
I moved the building instructions into building.md if that's what you meant. Did I miss something?